index.html

```diff
@@ -270,18 +272,19 @@ <h2>Core team</h2>
           <li><a href="http://simonjf.com/">Simon Fowler</a>.</li>
           <li><a href="https://homepages.inf.ed.ac.uk/vgalpin1/">Vashti Galpin</a>.</li>
           <li><a href="https://www.dhil.net/research/">Daniel Hillerstr&ouml;m</a>.</li>
-          <li><a href="http://rudi-horn.de/About">Rudi Horn</a>.</li>
           <li><a href="http://homepages.inf.ed.ac.uk/slindley/">Sam Lindley</a>.</li>
           <li><a href="http://www.wilmer-ricciotti.net/index.shtml">Wilmer Ricciotti</a>.</li>
-          <li><a href="http://ics.p.lodz.pl/~stolarek">Jan Stolarek</a>.</li>
+          <li><a href="http://https://thwfhk.github.io/">Wenhao Tang</a>.</li>
           <li><a href="http://homepages.inf.ed.ac.uk/wadler/">Philip Wadler</a>.</li>
         </ul>
         <h2>Alumni</h2>
         <ul>
           <li>Ezra Cooper.</li>
           <li>Gilles Dubochet.</li>
           <li><a href="http://stefan-fehrenbach.net/">Stefan Fehrenbach</a>.</li>
+          <li><a href="http://rudi-horn.de/About">Rudi Horn</a>.</li>
           <li><a href="https://jgbm.github.io/">Garrett Morris</a>.</li>
+          <li><a href="http://ics.p.lodz.pl/~stolarek">Jan Stolarek</a>.</li>
```

Comment:
Janek's web page has moved:

```suggestion
          <li><a href="https://jstolarek.github.io/">Jan Stolarek</a>.</li>
```